src/pages/groupComparison/VennSimple.tsx

```diff
@@ -22,12 +22,13 @@ export interface IVennSimpleProps {
     groups: {
         uid: string;
         cases: string[];
+        color: string;
+        nameWithOrdinal: string;
     }[]; // either 2 or 3
     x:number;
     y:number;
     width:number;
     height:number;
-    uidToGroup: { [uid: string]: ComparisonGroup };
```

Comment:
same comment as before. I don't get why this is preferred - this way we have to keep changing the prop whenever we add something to the group. I think it's best to just leave it as uidToGroup and keep the prop simple - pass in uid and cases.